docs/website/docs/getting-started/quickstart.md

```diff
@@ -1,163 +1,226 @@
+Push the component to the cluster:
 ```shell
-odo catalog list services
+odo push
 ```
 
+The URL will be listed in the `odo push` output, or can be found in `odo url list`.
+
+Browse the site and try it out! Note that you will not be able to add, remove or list the to-dos yet, as we have not linked the frontend and the backend components yet.
+
+## Create the backend Java component
+
+The backend application is a Java Spring Boot based REST API which will list, insert and delete to-dos from the database.
+
+Find `java-springboot` in the catalog:
 ```shell
-$ odo catalog list services
-Services available through Operators
-NAME                        CRDs
-postgresoperator.v5.0.3     PostgresCluster
+odo catalog list components
 ```
 
-If you don't see the Postgres Operator here, it might be still installing. Take a look at what you see in the `PHASE` column in below output:
+Example output of `odo catalog list components`:
 ```shell
-kubectl get csv
+Odo Devfile Components:
+NAME                             DESCRIPTION                                                         REGISTRY
+java-quarkus                     Quarkus with Java                                                   DefaultDevfileRegistry
+java-springboot                  Spring Boot® using Java                                             DefaultDevfileRegistry
+java-vertx                       Upstream Vert.x using Java                                          DefaultDevfileRegistry
+...
 ```
 
+Let's create the component below:
 ```shell
-$ kubectl get csv                         
-NAME                      DISPLAY                           VERSION   REPLACES                  PHASE
-postgresoperator.v5.0.3   Crunchy Postgres for Kubernetes   5.0.3     postgresoperator.v5.0.2   Succeeded
+cd ../backend
+odo create java-springboot backend
+odo url create --port 8080 --host <CLUSTER-HOSTNAME>.nip.io
+odo push
 ```
 
-If the `PHASE` is something other than `Succeeded`, you won't see it in `odo catalog list services` output, and you won't be able to create a working Operator backed service out of it either.
+Note, you will not be able to access `http://<YOUR-URL>/api/v1/todos` yet until we link the backend component to the database service.
 
-Now create the service using:
+## Create the Postgres service
 
+Use `odo catalog list services` to list all available operators.
 
-```sh
-odo service create --from-file ../postgrescluster.yaml
+By default, [Operator Lifecycle Manager (OLM)](/docs/getting-started/cluster-setup/kubernetes#installing-the-operator-lifecycle-manager-olm) includes no Operators and they must be installed via [Operator Hub](https://operatorhub.io/)
+
+Install the [Postgres Operator](https://operatorhub.io/operator/postgresql) on the cluster:
+```shell
+kubectl create -f https://operatorhub.io/install/postgresql.yaml
+```
```

Comment:
Although it works for the OpenShift users as well, they don't need to follow the CLI steps. They can do it via the web UI.

Reply:
Yeah.. Unfortunately I want to try and remain Kubernetes neutral and not mention OpenShift much. I was thinking of adding a note saying that "Alternatively, you can install it via the OpenShift UI, but I wasn't entirely sure.. 

Any suggestions?

Reply:
Let's keep it k8s specific in that case. OpenShift users can find docs targeted for OCP on docs.openshift.com.